### PR TITLE
Added a more basic example as the template ones are tough to use as s…

### PR DESCRIPTION
**Pull request by scstraus**
…tarting points

This is a real one from my config which I think will be easier for beginners.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/rest_command.markdown [hunk 1]
```diff
@@ -71,8 +71,30 @@ service_name:
 
 ## Examples
 
+### Basic example which uses PUT method and payload encoded as form data
+
+This example implements 2 rest commands to add service calls the missing shuffle functionality from the iTunes integration with a REST command.
```
Comment (frenck) [inline source/_integrations/rest_command.markdown]:
```suggestion
This example implements 2 REST commands to add service calls for the missing shuffle functionality of the iTunes integration.
```

File: source/_integrations/rest_command.markdown [hunk 2]
```diff
@@ -71,8 +71,30 @@ service_name:
...
+
+```yaml
+rest_command:
+  shuffle_on: 
+    url: "http://[YOUR_ITUNES-API_SERVER_IP]:8181/shuffle"
```
Comment (frenck) [inline source/_integrations/rest_command.markdown]:
```suggestion
    url: "http://YOUR_ITUNES-API_SERVER_IP:8181/shuffle"
```

File: source/_integrations/rest_command.markdown [hunk 3]
```diff
@@ -71,8 +71,30 @@ service_name:
...
+    method: put
+    content_type: "application/x-www-form-urlencoded"
+    payload: "mode=songs"
+  shuffle_off: 
+    url: "http://[YOUR_ITUNES-API_SERVER_IP]:8181/shuffle"
```
Comment (frenck) [inline source/_integrations/rest_command.markdown]:
```suggestion
    url: "http://YOUR_ITUNES-API_SERVER_IP:8181/shuffle"
```

File: source/_integrations/rest_command.markdown [hunk 4]
```diff
@@ -71,8 +71,30 @@ service_name:
...
+    method: put
+    content_type: "application/x-www-form-urlencoded"
+    payload: "mode=off"
+```
+
+### Using templates to change the payload based on entities
+
 The commands can be dynamic, using templates to insert values of other entities. Service call support variables for doing things with templates.
 
+In this example entry, you can see some simple [templates](/docs/configuration/templating/) in use for dynamic parameters.
```
Comment (frenck) [inline source/_integrations/rest_command.markdown]:
```suggestion
In this example, uses [templates](/docs/configuration/templating/) for dynamic parameters.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @scstraus!

Left some small comments, could you take a look? Thanks 👍

**Comment by scstraus** [comment]
All agreed. Thanks!

**Comment by frenck** [comment]
@scstraus I see you've resolved the comments, but I don't see a commit? Did you forget to push the commit?

**Comment by scstraus** [comment]
Sorry, I suck at Git. Are we good?

**Review by frenck — APPROVED** [review]
Thanks, @scstraus 👍

**Pull request merged**